samples/app-dev/Connectors/Miscellaneous/ODataS4hana/readme.md

```diff
@@ -0,0 +1,253 @@
+![Sample Response](../../../import-screenshots/ODataS4Hana/37.png)
+
+Provide OData URI value for which you want to delete data from SAP S4 Hana service. Here we are deleting updated ProductDescription property value using ODataURI: "A_ProductDescription(Product='A001',Language='EN')"
+![Sample Response](../../../import-screenshots/ODataS4Hana/38.png)
+
+Provide value for requestBody
+![Sample Response](../../../import-screenshots/ODataS4Hana/39.png)
+
+Mapp x-csrf-token value of ODataQuery activity to the input of the x-csrf-token header of ODataModifyDelete activity. Similarly mapp set-cookie value of ODataQuery activity to the input of the Cookie header of ODataModifyDelete activity. Set Content-Type value as "application/json".
+![Sample Response](../../../import-screenshots/ODataS4Hana/40.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/41.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/42.png)
+
+Provide Json Response Schema in Output Settings tab.
+![Sample Response](../../../import-screenshots/ODataS4Hana/43.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/44.png)
+
+Also in flow we have Log Message for every activity output and in last Return Activity for getting all activity output.
+
+![Sample Response](../../../import-screenshots/ODataS4Hana/45.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/46.png)
+
+### Run the application
+For running the application, first you have to push the app and then scale up the app.
+Then after sometime you can see your app goes in running state.
+
+![Sample Response](../../../import-screenshots/ODataS4Hana/47.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/48.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/49.png)
+
+Once your app reaches to Running state, go to Endpoints and for get_allcrud, select 'Try it Out’ option and then click on execute.
+
+Another option, If you want to test the sample in the Flow tester then follow below instructions:
+ 
+In flow, click on Test Button -> create Launch configuration -> click Next button -> click on Run
+
+![Sample Response](../../../import-screenshots/ODataS4Hana/50.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/51.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/52.png)
+
+## Outputs
+
+1. Flow Tester
+
+![Sample Response](../../../import-screenshots/ODataS4Hana/53.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/54.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/55.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/56.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/57.png)
+
+2. When hit endpoints
+
+![Sample Response](../../../import-screenshots/ODataS4Hana/58.png)
+![Sample Response](../../../import-screenshots/ODataS4Hana/59.png)
+
+
+## Troubleshooting
+
+* If you do not see the Endpoint enabled, make sure your apps is in Running status.
```

Comment:
Add one more point in this 
If you see test connection failed in connection tab, then check if Username and Password are correct